packages/menu/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "pcln-menu",
+  "version": "3.0.0",
```

Comment:
Should we set it to 4 to match the current major version?

Reply:
I thought it would bump to 4 with the major bump, is that the standard with new packages?

Reply:
Ohhh my bad, you are correct!